### DISCUSSION TITLE
日志全屏显示使其他标签页无法切换导航

### DISCUSSION THREAD

**Discussion by adsl156** [1Panel-dev/1Panel#5828] [Q&A]
### 联系方式

frank19961111@gamil.com

### 1Panel 版本

1.10.11-lts

### 问题描述

日志全屏显示使其他标签页无法切换导航

### 重现步骤

打开一个标签页面，并点击全屏显示，另一个显示器上随便打开一个面板功能，刷新页面后不显示左侧导航
![截图 2024-07-15 19-44-33](https://github.com/user-attachments/assets/23027015-b580-461f-b8ec-349f89ce74b8)

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。
